.gitignore

```diff
@@ -21,7 +21,7 @@ target
 /esvm
 .htpasswd
 .eslintcache
-plugins
+/plugins
```

Comment:
Love how I dragged my feet about your suggestion, and then didn't actually apply it. updated.